vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/sanity.go

```diff
@@ -120,6 +120,8 @@ type Config struct {
 	RemoveStagingPathCmd string
 	// Timeout for the executed commands for path removal.
 	RemovePathCmdTimeout int
+	// Timeout for the executed commands for path removal.
+	IdempotentCount int
```

Comment:
Cut-and-paste error: comment isn't updated.

Reply:
The semantic of that field needs to be explained, in particular the special values (0 and -1).